src/overlays/Modal.stories.tsx

```diff
@@ -254,30 +254,6 @@ export const ScrollableModalMinimalContent = () => {
   )
 }
 
-export const TransparentOverlayModal = () => (
```

Comment:
Talking with Emily, we determined this story was not representative or realistic so it's removed here